### PR TITLE
OLS-419: User feedback HTTP responses

### PR DESCRIPTION
**Pull request by tisnik**
## Description

User feedback HTTP responses

## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #[OLS-419](https://issues.redhat.com//browse/OLS-419)
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: ols/app/endpoints/feedback.py [hunk 1]
```diff
@@ -102,7 +126,18 @@ def store_user_feedback(
...
+    except Exception as e:
+        logger.error("Error storing user feedback")
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail={
+                "response": "Error storing user feedback",
+                "cause": str(e),
+            },
+        )
+        logger.error(e)
```
Comment (bparees) [inline ols/app/endpoints/feedback.py:141]: @tisnik does this statement actually get executed, given you raised an exception in the previous line?

also why not just log the error as part of the earlier logger.error message?  "error storing feedback: <error message>" seems more helpful.

Comment (tisnik) [reply ols/app/endpoints/feedback.py:141]: yes, it is leftover, let's update it

**Pull request merged**